main.py

```diff
@@ -91,8 +91,8 @@ def is_allowed_device(device: Device) -> bool:
             supported_commands = set()
 
             for command in commands:
-                bot_command = self.hubitat.he_to_bot_commands.get(command, None)
-                if bot_command:
+                if command in self.hubitat.he_to_bot_commands:
```

Comment:
Something is wrong in this and the next lines.

The next line uses `[]` and I assume that `he_to_bot_commands` is a dict.  This line does `foo in dict` that is wrong, one usually searches in keys or values

Reply:
Am I missing something or misunderstanding your comment?
```
>>> test = { "foo": "bar" }
>>> "foo" in test
True
>>> "bar" in test
False
>>> "foobar" in test
False
>>> test["foo"]
'bar'
```

Reply:
Well, this is news for me.  No, you code makes sense.  Python behavior used to be - do `in for .keys()` Looks like this is the default now for in operator.  

Your code is perfect!